logging/README.md

```diff
@@ -32,6 +32,8 @@ Before deploying, you must perform these tasks:
 - [create a local copy of the repository](#log_loc_copy)
 - [customize your deployment](#log_custom)
 
+Note: If you are deploying in a highly secure environment (in which pods cannot connect to the internet and network traffic is routed through a proxy server), you must add `localhost` to the list of hosts that are not routed through the proxy server. If you do not, the Elasticsearch deployment fails.  
```

Comment:
I'm not sure the pod connectivity issue is relevant in this particular case.  I think the use of the proxy server is the important restriction.  And, to be honest, although I could be wrong about this, I suspect most of the time "localhost" has already been added to the proxy list.  I think this particular customer just happened to have omitted it.  So, I wonder if we should phrase it as "_...be sure 'localhost' has been added to the lists..._" rather than telling them to do it.  But I may be reading more into the wording than is there.  If you want to provide more context/explanation, you could add that the deployment scripts use port forwarding to make REST API calls to Elasticsearch and Kibana as the scripts execute.

Reply:
Revised as noted.